packages/grid/_modules_/grid/components/containers/GridRootStyles.ts

```diff
@@ -1,13 +1,13 @@
 import { darken, lighten, Theme } from '@mui/material/styles';
 import { makeStyles } from '@mui/styles';
-import { getThemePaletteMode, muiStyleAlpha, createTheme } from '../../utils/utils';
+import { muiStyleAlpha, createTheme } from '../../utils/utils';
 import { gridClasses } from '../../gridClasses';
 
 const defaultTheme = createTheme();
 export const useStyles = makeStyles(
   (theme: Theme) => {
     const borderColor =
-      getThemePaletteMode(theme.palette) === 'light'
+      theme.palette.mode === 'light'
         ? lighten(muiStyleAlpha(theme.palette.divider, 1), 0.88)
```

Comment:
`alpha` from `@mui/material/styles` can be used now.

```suggestion
        ? lighten(alpha(theme.palette.divider, 1), 0.88)
```